src/TaskList.test.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import * as requests from "./utils/utils";
+import { render, screen } from "@testing-library/react";
+import TaskList from "./components/TaskList";
+import { BrowserRouter } from "react-router-dom";
+import { act } from "react-dom/test-utils";
+
+describe("TaskList", () => {
```

Comment:
a test for an error in the request is missing - look into `mockRejectedValue` in comparison to the `mockResolvedValue` you're using.